### DISCUSSION TITLE
Telescope Control Fixes and Version Confusion

### DISCUSSION THREAD

**Discussion by kantn8r** [Stellarium/stellarium#3428] [Q&A] [answered]
I've had a couple of issues I've participated in relative to the Telescope Control plugin. Both issues were closed but I've not yet tested the Losmandy G11G with the weekly snapshot  [stellarium-23.2-75a6ab4-qt6-win64.exe]. Now that I will have a clear night tonight to test, the weekly snapshot is a different version. What version I should install and test? Should I test [stellarium-23.2-75a6ab4-qt6-win64.exe], the current snapshot, or 23.3?

I'm an end user so I have trouble following what's going on here on Github.

EDITED to say that I'm just going to install 23.3.

**Comment by alex-w** [comment, marked as answer]
The release 23.3 was published yesterday
